lib/cocoapods-rome/post_install.rb

```diff
@@ -50,6 +52,17 @@ def xcodebuild(sandbox, target, sdk='macosx')
   Pod::UI.puts "Copying frameworks to `#{destination.relative_path_from Pathname.pwd}`"
 
   destination.rmtree if destination.directory?
+
+  installer_context.umbrella_targets.each do |umbrella|
+    umbrella.specs.each do |spec|
```

Comment:
This purposefully doesn't uniq, since the spec consumers for the different subspecs can have different vendored files